.vscode/launch.json

```diff
@@ -23,7 +23,8 @@
         "ENV_PREFIX": "DC",
         "FLASK_DEBUG": "1",
         "ENABLE_MODEL": "true",
-        "GOOGLE_CLOUD_PROJECT": "datcom-website-dev"
+        "GOOGLE_CLOUD_PROJECT": "datcom-website-dev",
+        "ENABLE_EVAL_TOOL": "true",
```

Comment:
might want to rename this flag to something that's not eval tool specific since it looks like this is the flag used to enable vertex ai models?

Reply:
maybe have another config like "Flask with NL and VertexAI"?

Reply:
@shifucun ok if I change it to ENABLE_VERTEX_AI ?

Reply:
sg

Reply:
On second thought and post discussion with Bo, I reverted this change.  I plan to move this to NL server, so let Web Server's use of Vertex AI continue to be gated under EVAL tool.